packages/runtime/container-runtime/.eslintrc.cjs

```diff
@@ -66,13 +72,7 @@ module.exports = {
 		"unicorn/throw-new-error": "error",
 
 		// TODO:
-		// unicorn/consistent-function-scoping
 		// unicorn/no-negated-condition
```

Comment:
This rule is likely to create a fair amount of code churn. I'll save it for its own PR.